### PR TITLE
Update site for v2.1.1 and v2.2.0-alpha release notes

### PR DESCRIPTION
**Pull request by jseldess**

### CODE REVIEW

**Comment by jseldess** [comment]
@spencerkimball, can you please draft a release note for https://github.com/cockroachdb/cockroach/pull/30143?

**Comment by jseldess** [comment]
@spencerkimball, actually, I can go with the PR description if you're ok with something like `cache zone config values to avoid repetitive deserialization`.

**Comment by jseldess** [comment]
@benesch, I can't figure out how to get the JS in `v2.2/install-cockroachdb.md` to redirect properly to `v2.2/install-cockroachdb-<os>.html`. Can you help?

**Comment by jseldess** [comment]
@benesch, nevermind. I just had some dates wrong, I think.

File: releases/v2.2.0-alpha.20181119.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+- Added a [Kubernetes configuration](https://github.com/cockroachdb/cockroach/tree/master/cloud/kubernetes/bring-your-own-certs) that shows how to use certificates generated outside of the Kubernetes-orchestrated CockroachDB cluster. [#27921][#27921] {% comment %}doc{% endcomment %}
+- Added a [Fluentd configuration](https://github.com/cockroachdb/cockroach/blob/master/cloud/kubernetes/fluentd-configmap.yml) for external logging of a Kubernetes-orchestrated CockroachDB cluster. [#26685](https://github.com/cockroachdb/cockroach/pull/26685)
+
+### SQL language changes
+
+- The [`EXPERIMENTAL_RELOCATE`](../v2.2/experimental-features.html) statement no longer temporarily increases the number of replicas in a range more than one above the range's replication factor, preventing rare edge cases of unavailability. [#29684][#29684] {% comment %}doc{% endcomment %}
+- The output of [`SHOW JOBS`](../v2.2/show-jobs.html) now reports ongoing jobs first in start time order, then completed jobs in finished time order. The `running_status` column becomes NULL when the status cannot be determined. [#30917][#30917] {% comment %}doc{% endcomment %}
```
Comment (lhirata) [inline releases/v2.2.0-alpha.20181119.md]: NULL --> `NULL`

File: releases/v2.2.0-alpha.20181119.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.2.0-alpha.20181118.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.2.0-alpha.20181118.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.2.0-alpha.20181118.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+docker pull cockroachdb/cockroach:v2.2.0-alpha.20181118
```
Comment (lhirata) [inline releases/v2.2.0-alpha.20181119.md]: Add `$`

File: releases/v2.1.1.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+- CockroachDB now de-correlates and successfully executes many queries containing correlated `EXISTS` subqueries. Previously, these queries caused a de-correlation error. [#32026][#32026] {% comment %}doc{% endcomment %}
+- If [diagnostics reporting](../v2.1/diagnostics-reporting.html) is enabled, attempts to use `CREATE/DROP SCHEMA`, `DEFERRABLE`, `CREATE TABLE (LIKE ...)`, `CREATE TABLE ... WITH`, and the "fetch limit" parameter (e.g., via JDBC) will now be collected as telemetry to gauge demand for these currently unsupported features. Also, the name of SQL [built-in functions](../v2.1/functions-and-operators.html) will be collected upon evaluation errors. [#31638][#31638] {% comment %}doc{% endcomment %}
+
+### Bug fixes
+
+- Fixed a small memory leak when running distributed queries. [#31759][#31759]
+- The `confkey` column of `pg_catalog.pg_constraint` no longer includes columns that were not involved in the foreign key reference. [#31895][#31895]
+- The [cost-based optimizer](../v2.1/cost-based-optimizer.html) no longer sometimes chooses the wrong index for a scan because of incorrect selectivity estimation. [#32011][#32011]
```
Comment (lhirata) [inline releases/v2.1.1.md]: Remove "sometimes"

File: releases/v2.2.0-alpha.20181119.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+- The [`cockroach sql`](../v2.2/use-the-built-in-sql-client.html) and [`cockroach demo`](../v2.2/cockroach-demo.html) commands now recognize the commands `exit` and `quit` to terminate the shell. [#31915][#31915] {% comment %}doc{% endcomment %}
+- The `cockroach debug estimate-gc` command now allows users to specify TTL period, with a default of 24 hours. [#31402][#31402]
+
+### Admin UI changes
+
+- Improved the layout of the [**Cluster Overview**](../v2.2/admin-ui-cluster-overview-page.html) page for large clusters with many nodes and ranges. [#31512][#31512] {% comment %}doc{% endcomment %}
+- Added the current node ID to the [**Advanced Debugging**](../v2.2/admin-ui-debug-pages.html) page to help identify the current node when viewing the web UI through a load balancer. [#31835][#31835] {% comment %}doc{% endcomment %}
+- The "Non-Table Cluster Data" section of the [**Databases**](../v2.2/admin-ui-databases-page.html) page now includes all non-table data types. Previously, this section only showed Time Series data. [#31830][#31830] {% comment %}doc{% endcomment %}
```
Comment (lhirata) [inline releases/v2.2.0-alpha.20181119.md]: "Non-Table Cluster Data" section should be **Non-Table Cluster Data** section, per our Style Guide

File: releases/v2.2.0-alpha.20181119.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+- Fixed a bug that sometimes caused invalid results or an "incorrectly ordered stream" error with streaming aggregations. [#31825][#31825]
+- Fixed a mismatch between lookup join planning and execution, which could cause queries to fail with the error "X lookup columns specified, expecting at most Y". [#31792][#31792]
+- Prevented a performance degradation related to overly aggressive Raft log truncations that could occur during [`RESTORE`](../v2.2/restore.html) or [`IMPORT`](../v2.2/import.html) operations. [#31914][#31914]
+- [`EXPLAIN ANALYZE`](../v2.2/explain-analyze.html) plans no longer show the processor with ID 0's stats in the Response box. [#31941][#31941]
+- Fixed rare deadlocks during [`IMPORT`](../v2.2/import.html), [`RESTORE`](../v2.2/restore.html), or [`BACKUP`](../v2.2/backup.html). [#31963][#31963]
+- Fixed a panic caused by incorrectly encoded Azure credentials. [#31984][#31984]
+- The [cost-based optimizer](../v2.2/cost-based-optimizer.html) no longer sometimes chooses the wrong index for a scan because of incorrect selectivity estimation. [#31937][#31937]
```
Comment (lhirata) [inline releases/v2.2.0-alpha.20181119.md]: Remove "sometimes"

File: releases/v2.2.0-alpha.20181119.md [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+
+### Admin UI changes
+
+- Improved the layout of the [**Cluster Overview**](../v2.2/admin-ui-cluster-overview-page.html) page for large clusters with many nodes and ranges. [#31512][#31512] {% comment %}doc{% endcomment %}
+- Added the current node ID to the [**Advanced Debugging**](../v2.2/admin-ui-debug-pages.html) page to help identify the current node when viewing the web UI through a load balancer. [#31835][#31835] {% comment %}doc{% endcomment %}
+- The "Non-Table Cluster Data" section of the [**Databases**](../v2.2/admin-ui-databases-page.html) page now includes all non-table data types. Previously, this section only showed Time Series data. [#31830][#31830] {% comment %}doc{% endcomment %}
+
+### Bug fixes
+
+- Hash functions with NULL input now return NULL. [#29822][#29822]
```
Comment (lhirata) [inline releases/v2.2.0-alpha.20181119.md]: NULL -> `NULL`

File: releases/v2.2.0-alpha.20181119.md (hunk 1 above)
Comment (jseldess) [reply releases/v2.2.0-alpha.20181119.md]: Done.

File: releases/v2.2.0-alpha.20181119.md (hunk 2 above)
Comment (jseldess) [reply releases/v2.2.0-alpha.20181119.md]: Done.

File: releases/v2.1.1.md (hunk 3 above)
Comment (jseldess) [reply releases/v2.1.1.md]: Done.

File: releases/v2.2.0-alpha.20181119.md (hunk 4 above)
Comment (jseldess) [reply releases/v2.2.0-alpha.20181119.md]: Done.

File: releases/v2.2.0-alpha.20181119.md (hunk 5 above)
Comment (jseldess) [reply releases/v2.2.0-alpha.20181119.md]: Done.

File: releases/v2.2.0-alpha.20181119.md (hunk 6 above)
Comment (jseldess) [reply releases/v2.2.0-alpha.20181119.md]: Done.

File: releases/v2.1.1.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.1.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.1.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.1.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+docker pull cockroachdb/cockroach:v2.1.1
```
[the remark this answers is not in the document]
Comment (jseldess) [reply releases/v2.1.1.md]: Done.

**Pull request merged**